lighthouse-core/test/results/sample_v2.json

```diff
@@ -3416,5 +3416,144 @@
       "title": "Crawling and Indexing",
       "description": "To appear in search results, crawlers need access to your app."
     }
+  },
+  "localeLog": {
```

Comment:
this will get big, but maybe it can be an optional thing?

Reply:
Yah +1 to this being an option that's off by default.